internal/data/migrations.go

```diff
@@ -0,0 +1,33 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+package data
+
+import (
+	"embed"
+	_ "embed"
+	"log/slog"
+
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	"github.com/golang-migrate/migrate/v4/source/iofs"
+)
+
+//go:embed migrations/*
+var content embed.FS
+
+func RunMigrations(postgres_endpoint string) {
+	d, err := iofs.New(content, "migrations")
```

Comment:
I'd rather use a more descriptive name for the `d` variable. IMO the only context where they're okay is when defining member functions.